include/seqan3/range/views/minimiser_hash.hpp

```diff
@@ -73,7 +73,7 @@ struct minimiser_hash_fn
     constexpr auto operator()(urng_t && urange,
                               shape const & shape,
                               window_size const window_size,
-                              seed const seed = seed{0x8F3F73B5CF1C9ADE}) const
+                              seed const seed = seqan3::seed{0x8F3F73B5CF1C9ADE}) const
```

Comment:
It's a bit unfortunate/confusing that the types have the same names as the variables here...